.pre-commit-config.yaml

```diff
@@ -18,7 +18,7 @@ repos:
     pass_filenames: false
   - id: typecheck
     name: Typecheck
-    entry: make typecheck-fast
-    types: [python]
```

Comment:
`types` should remain.

Reply:
well, we have to handle it anyway to support `pyright` call from `Makefile` or otherwise using pyright without pre-commit.

If it's really easier, happy to switch to just `pre-commit`, if so we need to call `pyright` via `pre-commit` in `Makefile` and anywhere else.

Reply:
That is exactly what I'm doing: I'm calling `pre-commit run typecheck` from `make typecheck`

Reply:
okay, great.

is it easy to pass other arguments to pyright with `pre-commit` from the command line? Don't know what they are yet, but I could imagine we might want to.

Reply:
I’m not sure. But I propose we cross that bridge when we get there.

Reply:
great, seems good to me.